src/main/java/Models/User/UserDao.java

```diff
@@ -0,0 +1,81 @@
+package Models.User;
+
+import Utils.Database;
+
+import java.sql.*;
+
+public class UserDao {
+
+    public static void insertUser(User user) throws DuplicateEmailPresentException {
+        Connection con = Database.getInstance().getConnection();
+
+        try {
+            PreparedStatement ps = con.prepareStatement("INSERT INTO USERS (FIRSTNAME, SURNAME, EMAIL, HASHED_PASSWORD, SALT) VALUES (?, ?, ?, ?, ?)");
+            ps.setString(1, user.getFirstName());
+            ps.setString(2, user.getSurname());
+            ps.setString(3, user.getEmail());
+            ps.setBytes(4, user.getPassword());
+            ps.setBytes(5, user.getSalt());
+            ps.executeUpdate();
+
+            System.out.println("User inserted");
```

Comment:
Looked at this after the last PR, and` System.out.println("String");` is indeed the correct way to log. Android is just weird.

Reply:
I think you can choose different logging frameworks etc, but yeah sout is the usual quick print statement